### PR TITLE
Lint fixes

### PR DESCRIPTION
**Pull request by tunnell**
Codacy followup to address nonuniformity in code from today.

### CODE REVIEW

File: examples/lax_root_batch.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+"""Make ROOT file of all lax output
+"""
+import hax
+from lax.lichens import sciencerun0
+
+RUN_NUMBER = 3956
+PAX_VERSION = '6.4.2'
```
Comment (pdeperio) [inline examples/lax_root_batch.py:7]: make these input arguments?

File: examples/lax_root_batch.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+hax.init(experiment='XENON1T',
+         pax_version_policy=PAX_VERSION)
+
+DF = hax.minitrees.load(RUN_NUMBER,
+                        ['Fundamentals', 'Basics', 'TotalProperties'])
```
Comment (pdeperio) [inline examples/lax_root_batch.py]: need also ```Extended``` and ```Proximity```?

**Review by pdeperio — CHANGES_REQUESTED** [review]

File: examples/lax_root_batch.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+for cuts in [sciencerun0.AllEnergy(),
+             sciencerun0.LowEnergy()]:
+    df_temp = cuts.process(DF.copy())
+    new_columns = [column for column in df_temp.columns if column not in OLD_COLUMNS]
+
+    cuts_name = cuts.name()
+
+    df_temp.loc[:, new_columns].to_root('lax_%s_%d.root' % (cuts_name,
+                                                            RUN_NUMBER))
```
Comment (pdeperio) [inline examples/lax_root_batch.py:26]: Add second argument to give the tree a more meaningful name than "default", e.g. 
```
, 'cuts'
```

Comment (tunnell) [reply examples/lax_root_batch.py:26]: This didn't work for me when I tried...

**Review by tunnell — COMMENTED** [review]
Just added comments

File: examples/lax_root_batch.py (hunk 1 above)
Comment (tunnell) [reply examples/lax_root_batch.py:7]: Should just make a lax-batch command if you want... I don't think people are using this though and instead just using notebook with hax then writing ROOT file at end.  Maybe could remove this?

File: examples/lax_root_batch.py (hunk 3 above)
Comment (pdeperio) [reply examples/lax_root_batch.py:26]: sorry, you're using the other function. see: https://github.com/ibab/root_pandas#writing-root-files

File: examples/lax_root_batch.py (hunk 1 above)
Comment (pdeperio) [reply examples/lax_root_batch.py:7]: would be nice to have a ```lax-batch``` command for data+MC mass productions

**Review by pdeperio — APPROVED** [review]

**Pull request merged**